syft/serde/serde.py

```diff
@@ -50,6 +50,13 @@
 
 from syft.messaging import Plan
 from syft.messaging import Message
+from syft.messaging import CommandMessage
+from syft.messaging import ObjectMessage
+from syft.messaging import ObjectRequestMessage
+from syft.messaging import IsNoneMessage
+from syft.messaging import GetShapeMessage
+from syft.messaging import ForceObjectDeleteMessage
+from syft.messaging import SearchMessage
```

Comment:
I would prefer one single import:
`from syft import messaging`
And then use the other classes with the prefix (e.g. messaging.Plan instead of Plan)

Reply:
done